contracts/plugins/assets/RTokenAsset.sol

```diff
@@ -70,7 +80,7 @@ contract RTokenAsset is IAsset {
     /// @return {UoA/tok} The lower end of the price estimate
     /// @return {UoA/tok} The upper end of the price estimate
     function price() public view virtual returns (uint192, uint192) {
-        try this.tryPrice() returns (uint192 low, uint192 high) {
+        try this.tryPrice() returns (uint192 low, uint192 high, uint192) {
```

Comment:
This is a good example why `tryPrice()` doesn't need to be on the interface. It's an internal detail of the implementation of the contract.